beta/lib/routes/help.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+const readPkgUp = require('read-pkg-up');
+const chalk = require('chalk');
+const path = require('path');
+const fs = require('fs');
+const { helpText } = require('../utils');
+
+const Help = (app) => {
+  const { env } = app;
+
+  // determine if we are in a movable blueprint dir or a seperate dir
+  const cwd = process.cwd();
+  const { packageJson = {} } = readPkgUp.sync(cwd) || {};
+  const { movableInk = {} } = packageJson;
+
+  // no blueprint found so lets just display the default CLI help text
+  if (!movableInk.blueprint) {
+    helpText();
+    return;
+  }
+
+  const { blueprint = '', version = '@latest' } = movableInk;
+
+  const packagePath = path.join(cwd, `node_modules/${blueprint}/app/utils`);
```

Comment:
You may want to use [`require.resolve()`](https://nodejs.org/api/modules.html#modules_require_resolve_request_options) instead — that will properly follow the node module resolution algorithm (whereas this may not work if you're in a monorepo and `node_modules` is in a different directory)

Reply:
Does this also mean that `utils.js` and/or `utils/index.js` is the file where the the help text will exist? And those files have to export a `helpText` function?

Reply:
> You may want to use [`require.resolve()`](https://nodejs.org/api/modules.html#modules_require_resolve_request_options) instead — that will properly follow the node module resolution algorithm (whereas this may not work if you're in a monorepo and `node_modules` is in a different directory)

Got it, that makes sense.


> Does this also mean that `utils.js` and/or `utils/index.js` is the file where the the help text will exist? And those files have to export a `helpText` function?

It means `utils/index.js` and the idea is that all the blueprint generators will have the `helpText` function to display out additional help text. 

I was thinking there should actually be a check to ensure that method is available and if it isn't we just display the default CLI help text.